.travis.yml

```diff
@@ -144,6 +145,10 @@ install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
 
+    # Use this until https://github.com/sphinx-gallery/sphinx-gallery/pull/255
+    # is merged.
+    - if [[ $SETUP_CMD == *build_docs* ]]; then pip -q install git+https://github.com/pllim/sphinx-gallery.git@stat-iter-deprec#egg=sphinx-gallery; fi
```

Comment:
Note: This fixes the deprecation warning from Sphinx for sure. This hack will have to stay until my PR over at `sphinx-gallery` gets merged, which I have no control over. But feel free to suggest a more elegant hack.